src/lib/services/submit.services.ts

```diff
@@ -88,8 +88,8 @@ export const initUserProposal = async ({
 		}
 
 		const { data, ...metadata } = docMetadata;
-		const { title, url, motionText } = data;
-		const editableMetadata = { title, url, motionText };
+		const { title, url, motionText, summary, nodeProviderId } = data;
```

Comment:
Are you sure it should be called `summary`? When submitting a proposal, the field is named `summary`?

Maybe I already ask you in previous PR and we changed for `summary`. I forgot. It just feels generic.

Reply:
Yes we discussed this in a [previous PR](https://github.com/peterpeterparker/proposals.network/pull/8#discussion_r1488321167).

I think we agreed to keep it as summary to stay consistent with the docs. But indeed I find it too generic. As this summary tends to include 3 things:
1. Link to forum discussion about your node provider candidacy
2. Link to self declaration document on the IC wiki
3. SHA256 hash of self declaration document so others can verify it has not changed.

I am thinking to modify this component to be more fine grained and explicitly ask for the data above, and then build the "summary" behind the scenes. But this would be for a separate PR and I have added it as a TODO in this file.

Reply:
Thanks, I had forget. Let's keep "summary" then.